compiler/src/dotty/tools/dotc/config/Feature.scala

```diff
@@ -26,8 +26,21 @@ object Feature:
   val dependent = experimental("dependent")
   val erasedDefinitions = experimental("erasedDefinitions")
   val symbolLiterals: TermName = deprecated("symbolLiterals")
+  val macros = experimental("macros")
 
-/** Is `feature` enabled by by a command-line setting? The enabling setting is
+  val experimentalWarningMessage = "Experimental features may only be used with nightly or snapshot version of compiler."
```

Comment:
Maybe we should refine the message by saying that Scala 3 `macros` are not experimental.